packages/flutter/lib/src/widgets/spell_check.dart

```diff
@@ -328,3 +352,38 @@ void _addComposingRegionTextSpans(
     )
   );
 }
+
+/// Methods for showing spell check suggestions for misspelled words in toolbar:
```

Comment:
Nit: I think this won't show up on the docs site and it shouldn't be triple slash.